authservice/jwt/jwt.go

```diff
@@ -73,3 +74,7 @@ func ValidateToken(token string, secret string) (bool, error) {
 	// This means the token matches
 	return true, nil
 }
+
+func GetSecret() string {
+	return os.Getenv("JWT_SECRET")
```

Comment:
Great work @vinicio. The code works fine but for this, the user will have to set up the environment variable. Maybe we should think about setting the environment variables using a `.env` file. I will push these commits to your pull request and merge.